pom.xml

```
@@ -27,8 +20,88 @@
       <name>Zextras Artifact Manager-releases</name>
       <url>https://zextras.jfrog.io/artifactory/java-sdk</url>
     </repository>
+    <repository>
+      <id>default</id>
+      <name>Maven public</name>
+      <url>https://repo.maven.apache.org/maven2/</url>
+    </repository>
+    <repository>
+      <id>jitpack.io</id>
+      <name>Jitpack</name>
+      <url>https://jitpack.io</url>
+    </repository>
```

You don't need Jitpack as dependency downloader, but you need `the-lucky-gunslingers` repository for now.
```suggestion
    <repository>
      <id>the-lucky-gunslingers</id>
      <name>Zextras Artifact Manager-releases</name>
      <url>https://zextras.jfrog.io/artifactory/the-lucky-gunslingers-java-sdk</url>
    </repository>
```